src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToSubmitHandlerTest.java

```diff
@@ -111,22 +98,6 @@ public void willCopyThePreviewFileToTheInterlocDirectionDocument() {
         assertNull(response.getData().getGenerateNotice());
         assertNull(response.getData().getDateAdded());
 
-        assertEquals(expectedDocument, response.getData().getSscsInterlocDirectionDocument());
-        assertEquals(Collections.singletonList(SscsInterlocDirectionDocuments.builder().value(expectedDocument).build()), response.getData().getHistoricSscsInterlocDirectionDocs());
-    }
-
-    @Test
-    public void whenNoPreviewAndDirectionDocumentExistsWillSetHistoricDocuments() {
```

Comment:
would be good to have a test like this that has more than one of this documentType in the sscsDocuments.